lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -15,6 +15,130 @@ def air_loop_hvac_unoccupied_threshold
     return 0.05
   end
 
+  # Determine the economizer type and limits for the the PRM
+  # Defaults to 90.1-2007 logic.
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] air loop
+  # @param climate_zone [String] ASHRAE climate zone, e.g. 'ASHRAE 169-2013-4A'
+  # @return [Array<Double>] [economizer_type, drybulb_limit_f, enthalpy_limit_btu_per_lb, dewpoint_limit_f]
+  def air_loop_hvac_prm_economizer_type_and_limits(air_loop_hvac, climate_zone)
+    economizer_type = 'NoEconomizer'
+    drybulb_limit_f = nil
+    enthalpy_limit_btu_per_lb = nil
+    dewpoint_limit_f = nil
+
+    case climate_zone
```

Comment:
I suggest to create lists of CZs for each category and loop them to assign the value (see similar comment below). There is no 65 F category for Appendix G, all the missing CZs from the high-limit table do not get economizer.